index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel='stylesheet' href='styles.css'>
+    <title>Document</title>
+</head>
+<body>
+
+  <nav id='navbar'> 
+    <a href='#projects'>Projects</a>
+    <a href='#projects'>About</a>
+    <a href='#projects'>Contact</a>
+  </nav>
```

Comment:
It's better to group the elements of the navigation bar in `<ul>` and a `li` tag to wrap each of them would be better appropriate for semantic reasons.